pkg/controller/volume/persistentvolume/scheduler_bind_cache_metrics.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package persistentvolume
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"time"
+)
+
+// VolumeSchedulerSubsystem - subsystem name used by scheduler
+const VolumeSchedulerSubsystem = "volumeScheduler"
```

Comment:
Generated metric name will be `volumeScheduler_xxx`, e.g. `volumeScheduler_ volume_binder_cache_requests_total`. I suggest using snake case to be consistent with existing metrics. 

I recommend using `scheduler_volume`  as subsystem name, and removing common `volume_` prefix for each metric. Examples:

- subsystem: scheduler_volume
- metrics:
  - binder_cache_requests_total
  - scheduling_duration_microseconds
  - volume_binding_error_total

Full metric names will be:

- scheduler_volume_binder_cache_requests_total
- scheduler_volume_scheduling_duration_microseconds
- scheduler_volume_volume_binding_error_total

@msau42 @bsalamat What do you think?

Reply:
@cofyc  Great, thanks for your guidelines.